sdk/Trace/TracerProvider.php

```diff
@@ -43,6 +43,15 @@ class TracerProvider implements API\TracerProvider
         }
 
         $this->spanProcessors = $spanProcessors;
+
+        register_shutdown_function([$this, 'shutdown']);
+    }
+
+    public function shutdown(): void
+    {
+        foreach ($this->spanProcessors as $spanProcessor) {
+            $spanProcessor->shutdown();
+        }
```

Comment:
This `TracerProvider::shutdown()` helper is only needed while the TracerProvider is managing the array of `$spanProcessors`... once https://github.com/open-telemetry/opentelemetry-php/pull/94 is merged, this helper can go away and the callback registration in the constructor can look like:
`register_shutdown_function([$this->spanProcessors, 'shutdown'])` since it will handle the "fan-out" to all the other downstream SpanProcessors